test/kpod_mount.bats

```diff
@@ -8,6 +8,10 @@ function teardown() {
     cleanup_test
 }
 
+function setup() {
+    copy_images
```

Comment:
We only use `redis:alpine` in this file, so I don't think this saves it any time.